__tests__/pathToKebabCase.test.js

```diff
@@ -0,0 +1,13 @@
+const pathToKebabCase = require('../build.js').pathToKebabCase
+
+test('valid token', () => {
```

Comment:
```suggestion
it('returns a valid token name in kebab-case', () => {
```

also recommend wrapping these in a describe block too ideally...